### PR TITLE
handle flattening of cert-dir

### PR DESCRIPTION
**Pull request by deads2k**
Files are moving due to https://github.com/openshift/origin/pull/1737 and we have significant drift from reality.  This brings us back into alignment.

Do not merge until  https://github.com/openshift/origin/pull/1737 goes in.

### CODE REVIEW

**Comment by deads2k** [comment]
@liggitt

File: dev_guide/master_node_configuration.adoc [hunk 1]
```diff
@@ -17,16 +17,16 @@ The config files are fully specifying with no defaulting.  This means that any e
 == Create the starting config files
 The `openshift start` command accepts flags that indicate that it should simply write the config file that it would have used and terminate.  This is useful for getting a starting point for the config.  You can do this by running 
 
-* `openshift start --writeconfig --master-config=master.yaml --node-config=node.yaml`
-* `openshift start master --write-config --config=master.yaml`
-* `openshift start node --write-config --config=node.yaml`
+* `openshift start --writeconfig=openshift.local.config
```
Comment (liggitt) [inline dev_guide/master_node_configuration.adoc]: write-config

File: dev_guide/master_node_configuration.adoc [hunk 2]
```diff
@@ -17,16 +17,16 @@ The config files are fully specifying with no defaulting.  This means that any e
 == Create the starting config files
 The `openshift start` command accepts flags that indicate that it should simply write the config file that it would have used and terminate.  This is useful for getting a starting point for the config.  You can do this by running 
 
-* `openshift start --writeconfig --master-config=master.yaml --node-config=node.yaml`
-* `openshift start master --write-config --config=master.yaml`
-* `openshift start node --write-config --config=node.yaml`
+* `openshift start --write-config=openshift.local.config
+* `openshift start master --write-config=openshift.local.config/master`
+* `osadm create-node-config --node-dir=openshift.local.config/node-myhost --node=myhost --hostnames=myhost.local`
```
Comment (liggitt) [inline dev_guide/master_node_configuration.adoc]: maybe add an example IP to the --hostnames arg, just so people know they can (and probably should) include the node IP

File: getting_started/dev_get_started/installation.adoc [hunk 3]
```diff
@@ -51,8 +51,8 @@ $ sudo docker exec -it openshift-origin bash
 . Because OpenShift services are secured by TLS, clients must accept the server certificates and present their own client certificate. These certificates are generated when the master server is started. You must point `osc` and `curl` at the appropriate CA bundle and client key and certificate to connect to OpenShift. Set the following environment variables:
 +
 ----
-# export KUBECONFIG=/var/lib/openshift/openshift.local.certificates/admin/.kubeconfig
-# export CURL_CA_BUNDLE=/var/lib/openshift/openshift.local.certificates/admin/root.crt
+# export KUBECONFIG=/var/lib/openshift/openshift.local.config/master/admin.kubeconfig
```
Comment (liggitt) [inline getting_started/dev_get_started/installation.adoc]: envvar

File: getting_started/dev_get_started/installation.adoc [hunk 4]
```diff
@@ -104,9 +104,9 @@ $ ./openshift help
 . Because OpenShift services are secured by TLS, clients must accept the server certificates and present their own client certificate. These certificates are generated when the master server is started. You must point `osc` and `curl` at the appropriate CA bundle and client key and certificate to connect to OpenShift. Set the following environment variables:
 +
 ----
-$ export KUBECONFIG=`pwd`/openshift.local.certificates/admin/.kubeconfig
```
Comment (liggitt) [inline getting_started/dev_get_started/installation.adoc]: envvar

**Comment by liggitt** [comment]
a few comments, sweep for rogue KUBECONFIG references, LGTM otherwise

**Comment by liggitt** [comment]
pre-doc reorg :-/

**Comment by deads2k** [comment]
rebased.  I think that it found everything correctly.  I don't see any rogue `.crt`, or `.kubeconfig` file references.

**Comment by deads2k** [comment]
rebased.

**Comment by adellape** [comment]
@deads2k Thanks, will merge after the build passes.

**Pull request merged**